client/src/components/MultiJurisdictionAudit/useSetupMenuItems/_mocks/index.ts

```diff
@@ -49,7 +43,7 @@ export const auditSettings: {
     randomSeed: null,
     riskLimit: null,
     auditType: 'BATCH_COMPARISON',
-    auditMathType: 'BRAVO',
+    auditMathType: 'MACRO',
```

Comment:
@jonahkagan Just curious, what's the change here?

Reply:
just making the data more realistic. BRAVO can only be used with BALLOT_POLLING, and MACRO only with BATCH_COMPARISON